PRODUCT_MATRIX.md

```diff
@@ -0,0 +1,20 @@
+| Product | Product Key  | Package Name |
+| ------- | ------------ | ------------ |
+| Chef Client | chef | chef |
```

Comment:
I agree.  Package Name is really internal to the resource, and should be removed from the md.